content/blog/2017/2017-04-05-welcome-to-blue-ocean-editor.adoc

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Getting Started with Blue Ocean and the Pipeline Visual Editor"
```

Comment:
Suggest shortening to "Getting Started with Blue Ocean's Visual Pipeline Editor"